compaction/compaction_strategy.cc

```diff
@@ -87,6 +87,34 @@ std::optional<sstring> compaction_strategy_impl::get_value(const std::map<sstrin
     return it->second;
 }
 
+void compaction_strategy_impl::validate_options(const std::map<sstring, sstring>& options, std::map<sstring, sstring>& unchecked_options) {
```

Comment:
please update commit message to mention `compaction_strategy_impl::validate_options` rather than the Javaesque `compaction_strategy_impl::validateOptions`